cmake/kokkos_enable_options.cmake

```diff
@@ -47,6 +47,7 @@ KOKKOS_ENABLE_OPTION(DEBUG_BOUNDS_CHECK   OFF "Whether to use bounds checking -
 KOKKOS_ENABLE_OPTION(COMPILER_WARNINGS    OFF "Whether to print all compiler warnings")
 KOKKOS_ENABLE_OPTION(PROFILING            ON  "Whether to create bindings for profiling tools")
 KOKKOS_ENABLE_OPTION(PROFILING_LOAD_PRINT OFF "Whether to print information about which profiling tools got loaded")
+KOKKOS_ENABLE_OPTION(PROFILING_SOURCE_INFO OFF "Whether the names Kokkos includes in source files includes file/line info")
```

Comment:
where is this option used?

Reply:
@crtrott: used here https://github.com/kokkos/kokkos/pull/2891/files#diff-95625f44836818ba366d2d5f343f0d53R657

Reply:
I am also confused about where this is used.  Is it a new macro meant for user code?